src/components/loader-bar/loader-bar.stories.mdx

```diff
@@ -1,8 +1,7 @@
 import { useState } from "react";
 import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
-import LoaderBar from ".";
-import Button from "../button";
 import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import DefaultStory from "./loader-bar.stories.tsx";
```

Comment:
```suggestion
import * as stories from "./loader-bar.stories.tsx";
```
Since the stories from `stories.tsx` are named exports, we cannot import it like this unfortunately. We have to import them all like so then access them individually via `stories.<storyName>`